### PR TITLE
Arkansas household and dependents care credit

### LINKED ISSUE #2899: Arkansas household and dependent child care credit
law file:
https://casetext.com/regulation/arkansas-administrative-code/agency-006-department-of-finance-and-administration/division-05-division-of-revenues/regulation-1997-4-comprehensive-individual-income-tax-regulations/rule-26-51-502-household-and-dependent-care-services

### PR DESCRIPTION
**Pull request by Jingj-WANG**
### Summary
:sparkles::page_facing_up::bookmark:

This pull request implements the Arkansas child care expenses credit as a new feature in PolicyEngine US. It adds a new changelog entry, a new parameter file `decimal_amount_to_agi`, and updates the relevant formulas and tests.

> _`Arkansas` credit_
> _New parameter file added_
> _Fall leaves change with `AGI`_

### Walkthrough
*  Add a new credit for child care expenses in Arkansas ([link](https://github.com/PolicyEngine/policyengine-us/pull/2934/files?diff=unified&w=0#diff-5885042762833a4aef51405a4036067e4e6aefa7a5091a2987d1d9d1fc2647e8R1-R4), [link](https://github.com/PolicyEngine/policyengine-us/pull/2934/files?diff=unified&w=0#diff-a210bc5aed1c8b0af91110ad40fd977f0984382b4b011c0be8dac2abc89b5d87R1-R79))
  - Create a new parameter file `decimal_amount_to_agi` that defines the credit rate as a fraction of the federal adjusted gross income, with different brackets for different income levels ([link](https://github.com/PolicyEngine/policyengine-us/pull/2934/files?diff=unified&w=0#diff-a210bc5aed1c8b0af91110ad40fd977f0984382b4b011c0be8dac2abc89b5d87R1-R79))
  - Update the changelog file to reflect the new feature and the minor version bump ([link](https://github.com/PolicyEngine/policyengine-us/pull/2934/files?diff=unified&w=0#diff-5885042762833a4aef51405a4036067e4e6aefa7a5091a2987d1d9d1fc2647e8R1-R4))

fixes #2899

### CODE REVIEW

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/cdcc/Readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Child and Dependent Care Credit
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/cdcc/Readme.md:1]: README.md

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/cdcc/match.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+description: Arkansas matches this percentage of the federal child care credit.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/cdcc/match.yaml:1]:
```suggestion
description: Arkansas matches this percentage of the federal child and dependent care credit. 
```

File: policyengine_us/parameters/gov/states/ar/tax/income/credits/cdcc/match.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+description: Arkansas matches this percentage of the federal child care credit. 
+metadata:
+  unit: /1
+  period: year
+  label: Arkansas household and dependents care credit match
+  reference:
+    - title: Ark. Code R. § 26-51-502(c)(1)(A)(ii)
+      href: https://codes.findlaw.com/ar/title-26-taxation/ar-code-sect-26-51-502/
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions | line 35
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=15
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ar/tax/income/credits/cdcc/match.yaml:10]: Cite a reference that specifies 20%. References must always support parameter values.

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
add to parameter list of refundable credits

**Comment by PavelMakarchuk** [comment]
> add to parameter list of refundable credits

added.

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**